seed_data/management/programs.json

```diff
@@ -16,7 +16,11 @@
                         "edx_course_key": "course-v1:MITx+Digital+Learning+100+Jan_2015",
                         "title": "Digital Learning 100 - January 2015",
                         "start_date": "2015-01-15T00:00:00",
-                        "enrollment_end": "2015-01-29T00:00:00"
+                        "enrollment_end": "2015-01-29T00:00:00",
+                        "course_price": {
+                            "price": 1000,
+                            "is_valid": true
```

Comment:
i think we should get rid of all of these `is_valid` key-values and just set that value to True when we deserialize. we do this when we deserialize Programs (see `deserialize_program_data_list`). i know there are drawbacks when we set values in two different places, but it's safe to assume we want a specific value for this attribute when we're seeding the db (like we do with `Program.live`). also, we get a shorter json file